transportable-udfs-examples/settings.gradle

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 LinkedIn Corporation. All rights reserved.
+// Licensed under the BSD-2 Clause license.
+// See LICENSE in the project root for license information.
+
+rootProject.name = 'transportable-udfs-examples'
+
+includeBuild('../')
```

Comment:
Will our IDE understand this? Have we tried importing this examples project in Intellij and see it resolves the transport dependencies correctly?

Reply:
IDE (atleast IntelliJ Idea) will understand the `includeBuild` statement correctly. It will build the included build before building the examples project. I have verified this. If we open the examples project in the IDE, it will also automatically import the core project and we are able to work on both projects simultaneously (atleast IntelliJ Idea).
![image](https://user-images.githubusercontent.com/6961317/54716673-7cb1ce00-4b13-11e9-8df0-d1bbb8fa8651.png)

Reply:
Thanks for checking @shardulm94